modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/SystemObjectRelatedObjectEntriesTest.java

```diff
@@ -256,15 +256,86 @@ public void testGetManyToManySystemObjectRelatedObjectEntries()
 	public void testGetManyToOneSystemObjectRelatedObjectEntries()
```

Comment:
Maybe we can simplify this:
```java
	@Test
	public void testGetManyToOneSystemObjectRelatedObjectEntries()
		throws Exception {

		String pkObjectFieldName = _objectDefinition.getPKObjectFieldName();

		String objectDefinitionNameReference = pkObjectFieldName.substring(
			0, pkObjectFieldName.length() - 2);

		ObjectRelationship objectRelationship =
			ObjectRelationshipTestUtil.addObjectRelationship(
				_objectDefinition, _userSystemObjectDefinition,
				_user.getUserId(),
				ObjectRelationshipConstants.TYPE_ONE_TO_MANY);

		_objectRelationships.add(objectRelationship);

		String relatedObjectEntryFieldBase = StringBundler.concat(
			"r_", objectRelationship.getName(), "_",
			objectDefinitionNameReference);

		_testGetManyToOneSystemObjectRelatedObjectEntries(StringPool.BLANK, 0, relatedObjectEntryFieldBase, _user.getUserId());

		_testGetManyToOneSystemObjectRelatedObjectEntries(StringPool.BLANK, 0, relatedObjectEntryFieldBase, _userAccountJSONObject.getLong("id"));

		ObjectRelationshipLocalServiceUtil.
			addObjectRelationshipMappingTableValues(
				_user.getUserId(), objectRelationship.getObjectRelationshipId(),
				_objectEntry.getPrimaryKey(), _user.getUserId(),
				ServiceContextTestUtil.getServiceContext());

		_testGetManyToOneSystemObjectRelatedObjectEntries(_objectEntry.getExternalReferenceCode(), _objectEntry.getObjectEntryId(), relatedObjectEntryFieldBase, _user.getUserId());
                
		ObjectRelationshipLocalServiceUtil.
			addObjectRelationshipMappingTableValues(
				_user.getUserId(), objectRelationship.getObjectRelationshipId(),
				_objectEntry.getPrimaryKey(),
				_userAccountJSONObject.getLong("id"),
				ServiceContextTestUtil.getServiceContext());

		JSONObject jsonObject = HTTPTestUtil.invokeToJSONObject(
			null, _getLocation(objectRelationship.getName()), Http.Method.GET);

		Assert.assertNotNull(jsonObject.get(objectRelationship.getName()));
	}

	private void _testGetManyToOneSystemObjectRelatedObjectEntries(String expectedObjectEntryExternalReferenceCode, long expectedObjectEntryId,
		String relatedObjectEntryFieldBase, long userId) throws Exception {

		JSONObject jsonObject = HTTPTestUtil.invokeToJSONObject(
			null,
			StringBundler.concat(
				_getLocation(), StringPool.SLASH, userId),
			Http.Method.GET);

		Assert.assertEquals(
			expectedObjectEntryExternalReferenceCode,
			jsonObject.get(relatedObjectEntryFieldBase + "ERC"));
		Assert.assertEquals(expectedObjectEntryId,
			jsonObject.getLong(
				relatedObjectEntryFieldBase + "Id"
			));
	}
```
From my point of view, I prefer to use the Relationship Endpoints instead of the `ObjectRelationshipLocalServiceUtil` because we are assuring that everything is working properly. What do you think?